enhancements/network/user-defined-network-segmentation.md

```diff
@@ -565,6 +566,13 @@ type ClusterUserDefinedNetwork struct {
 
 // ClusterUserDefinedNetwork defines the desired state of ClusterUserDefinedNetwork.
 type ClusterUserDefinedNetwork struct {
+    // VRF is an optional VRF device name to use for the network. It has to be less than
+    // 16 characters long and unique. If omitted, the ClusterUserDefinedNetwork name
```

Comment:
> // VRF is an optional VRF device name to use for the network. It has to be less than
    // 16 characters long and unique.

Asking to understand, what will happen if VRF is specified? 
Does it mean the pod is wired to some device in the node?
If so, does it fit localnet topology?

Reply:
No, it just means that name will be used instead of the currently generated one.